bin/lib/zopen-build

```diff
@@ -840,6 +843,11 @@ resolveCommands()
     export ZOPEN_INSTALL_CMD="${ZOPEN_INSTALL} ${ZOPEN_INSTALL_OPTS}"
     paxFileName="${ZOPEN_NAME}.${LOG_PFX}.zos.pax.Z"
     export ZOPEN_PAX_CMD="pax -w -z -x pax \"-s#${ZOPEN_INSTALL_DIR}/#${ZOPEN_NAME}.${LOG_PFX}.zos/#\" -f \"${paxFileName}\" \"${ZOPEN_INSTALL_DIR}/\""
+    PROJECT_NAME="$(echo ${ZOPEN_NAME} | cut -d'-' -f1)"
+    if [ "${PROJECT_NAME}" != "${ZOPEN_NAME}" ]; then
+      linkPath="$(dirname ${ZOPEN_INSTALL_DIR})/${PROJECT_NAME}"
+      export ZOPEN_LINK_CMD="rm -f ${linkPath} && ln -s ${ZOPEN_INSTALL_DIR} ${linkPath}"
```

Comment:
would it make sense to verify it's a symbolic link before removing it, in case it is a directory of stuff?

Reply:
Should we introduce a --clobber option to clobber the symlink/ZOPEN_INSTALL_DIR?

Reply:
I've added a check for now